src/html5_playback.js

```diff
@@ -27,18 +29,38 @@ export default class HTML5TVsPlayback extends Playback {
 
   get tagName() { return 'video' }
 
+  get mediaType() { return this.el.duration === Infinity ? Playback.LIVE : Playback.VOD }
+
   get isReady() { return this.el.readyState >= READY_STATE_STAGES.HAVE_CURRENT_DATA }
 
   get playing() { return !this.el.paused && !this.el.ended }
 
   get currentTime() { return this.el.currentTime }
 
-  get duration() { return this.el.duration }
+  get duration() {
+    // The HTMLMediaElement.duration returns Infinity for live streams: https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/duration#value
+    return this.isLive
+      ? this.el.seekable.end(0) - this.el.seekable.start(0)
+      : this.el.duration
+  }
 
   get ended() { return this.el.ended }
 
   get buffering() { return this._isBuffering }
 
+  get isLive() { return this.mediaType === Playback.LIVE }
+
+  get minimumDvrSizeConfig() {
+    const DVRSizeConfig = this.options.playback && this.options.playback.minimumDvrSize
```

Comment:
Since `DVRSizeConfig` is a number,  I think `dvrSizeConfig` would be a clearer name.  `UpperCamelCase` is usually associated to classes or enums and this is not the case here.

Reply:
Done here: 642a7b5